infertrade/data/simulate_data.py

```diff
@@ -56,4 +56,4 @@ def simulated_correlated_equities_4_years_gen():
     independent_asset = np.cumprod([1 + 0.02 * (np.random.random() - np.random.random()) for _ in range(1000)])
     asset_2 = asset_1 * independent_asset
 
-    return pd.DataFrame({PandasEnum.PRICE.value: asset_1, PandasEnum.SIGNAL.value: asset_2,})
+    return pd.DataFrame({PandasEnum.MID.value: asset_1, PandasEnum.SIGNAL.value: asset_2,})
```

Comment:
PandasEnum.PRICE.value was introduced in this line in commit 7e35fdf713373a86bb13471ee30ca1c0853a498a , but apparently that values was never part of PandasEnum.